test/wuschel.jl

```diff
@@ -14,7 +14,7 @@ m = myread(s)
 
 rs = ReactionSystem(m)
 @test isequal(length(rs.eqs), 4139)
-@test isequal(length(rs.species), 1265)
+@test isequal(length(rs.states), 1265)
```

Comment:
for the future, use the accessor functions https://mtk.sciml.ai/dev/basics/AbstractSystem/#Composition-and-Accessor-Functions
its totally fine here but it might give a warning. really no big deal

Reply:
the reason for such a warning is in the case of subsystems just FYI